modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -43,14 +43,16 @@
 #define OPENCV_OPENCL_HPP
 
 #include "opencv2/core.hpp"
+#include <typeinfo>
+#include <typeindex>
 
 namespace cv { namespace ocl {
 
 //! @addtogroup core_opencl
 //! @{
 
 CV_EXPORTS_W bool haveOpenCL();
-CV_EXPORTS_W bool useOpenCL();
+CV_EXPORTS_W bool useOpenCL(bool init = true); // TODO: can we change function signature?
```

Comment:
@alalek Should be changed as we can't change signature of existent function?

Reply:
@asmorkalov, this PR has 4 failing tests for vp9 decoding on [Windows 10](https://build.opencv.org.cn/job/precommit/job/windows10/1294/), is it same system setup and probably same reason as [issue 19694](https://github.com/opencv/opencv/issues/19694)?